.standard.yml

```diff
@@ -0,0 +1,6 @@
+plugins:
+  - standard-rails
+ignore:
+  - 'lib/log/logger.rb':
+      - Rails/EnvironmentVariableAccess
+      - Style/GlobalStdStream
```

Comment:
This ignore rule works, or `bundle exec standardrb` would complain. However, the previous line doesn't work - it complains regardless.